src/handlers/directTransition.js

```diff
@@ -48,6 +48,9 @@ export const handler = async (
       throw new Error('directTransition: ERROR: No capacity to make a call');
     }
 
+    // CR: Mickey: made a note on `incrementUsedCount`, but do we have to consider if
+    //    handler makes multiple calls to the vendor api for a single message
+
```

Comment:
@MickeyMcc Good catch. In the next iteration, I will add a parameter to query the worker in advance or set some other env variable to mention the number of vendor calls that will be made for every type of event and increment the usedCount based on that.